cmd/otelcol/main.go

```diff
@@ -19,7 +19,7 @@ package main
 import (
 	"log"
 
-	"github.com/open-telemetry/opentelemetry-collector/defaults"
+	"github.com/open-telemetry/opentelemetry-collector/component/componentdefault"
```

Comment:
`component/component*/` stutters. Maybe `component/defaultcomponents` instead? Neither looks good to me, maybe you have other ideas.

Or alternatively let's move this to `service/defaultcomponents`. I think it fits better than under `components` directory.

What do you think?

Reply:
We do use a lot this pattern `component/component*/` so it is consistent.

I do agree that I think this are the default components for the service so having them there makes more sense.